test/_common.py

```diff
@@ -53,7 +53,7 @@
 _item_ident = 0
 
 # OS feature test.
-HAVE_SYMLINK = hasattr(os, 'symlink')
+HAVE_SYMLINK = sys.platform != 'win32'
```

Comment:
I'm comitting it now because I think we're just missing an appropriate error message, but otherwise the functionality is ok